api/src/main/java/io/minio/MakeBucketArgs.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+public class MakeBucketArgs {
```

Comment:
1. You need to have `newBuilder()` which returns `MakeBucketArgs.Builder`
2. `Builder()` doesn't create another `Builder`
3. Do not assume default region is `us-east-1`. The default region is the one passed to `MinioClient` or `us-east-1`. Here you don't have much control.
4. Refer https://square.github.io/okhttp/3.x/okhttp/okhttp3/Headers.html for clarity.